docs/development/local-development.md

```diff
@@ -1,13 +1,14 @@
 # Local Development
 
-This document serves to give tips and tricks on how to run Renovate locally to add features or fix bugs.
-Please submit PRs to improve it if you think anything is unclear or you can think of something that should be added.
+This document gives tips and tricks on how to run Renovate locally to add features or fix bugs.
+You can improve this documentation by opening a pull request.
+For example, if you think anything is unclear, or you think something needs to be added, open a pull request!
 
-## Install
+## Installation
 
 #### Prerequisites
```

Comment:
```suggestion
### Prerequisites
```

should be h3 ? 🤔

Reply:
The original docs jump from `h2` to `h4` as well, so I kept it that way. But we can change this now of-course... :smile: